src/mixins/object_origin.mixin.js

```diff
@@ -129,16 +129,28 @@
     },
 
     /**
-     * Returns the point in local coordinates
-     * @param {fabric.Point} point The point relative to the global coordinate system
+     * Returns the normalized point (rotated relative to center) in local coordinates
+     * @param {fabric.Point} point The point relative to canvas coordinate system
      * @param {String} originX Horizontal origin: 'left', 'center' or 'right'
      * @param {String} originY Vertical origin: 'top', 'center' or 'bottom'
      * @return {fabric.Point}
      */
-    toLocalPoint: function(point, originX, originY) {
-      var center = this.getCenterPoint(),
-          p, p2;
+    toLocalPoint: function (point, originX, originY) {
```

Comment:
I think this method isn't used anywhere. And I'm not sure it correct.

Reply:
well if is not used anywhere now we can trash it.

Reply:
d410e0da

Reply:
aacaa76a